src/Aion/PublicKey.h

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+namespace TW {
+namespace Aion {
+
+class PublicKey {
```

Comment:
Why do we need a new implementation of `PublicKey`?

Reply:
This is temporary as well, made it because of a different size.